templates/payment/form-atome.php

```diff
@@ -9,7 +9,7 @@
 
         <p id="omise_atome_phone_field" class="form-row form-row-wide" style="display: none;">
             <span class="woocommerce-input-wrapper">
-                <input id="omise_phone_number" class="input-text" name="omise_phone_number" type="tel" autocomplete="off">
+                <input id="omise_atome_phone_number" class="input-text" name="omise_atome_phone_number" type="tel" autocomplete="off">
```

Comment:
Can we set initial value of Atome phone number from billing phone number?
Since there is no label, can we also add placeholder?